bash/asm

```diff
@@ -5,25 +5,50 @@
 # debug flag; 0=no debug, 1=debug
 debug=0
 
-ECHO=0
-if [ "$1" = "tron" ] || [ "$1" = "TRON" ]; then
-#    echo "setting echo to 1"
-	ECHO=1
-	shift
+rc=0
+trace=0
+
+if [ $# -gt 0 ]; then
+    # save current nocasematch setting
+    SHELLNOCASEMATCH=$(shopt -p nocasematch)
+    shopt -s nocasematch
+    case $1 in
+        "tron")
+            shift
+            trace=1
+            ;;
+        "troff")
+            shift
+            trace=0
+            ;;
+    esac
+    # restore nocasematch setting
+    $SHELLNOCASEMATCH
```

Comment:
This logic is repeated in many of the modules -- just a thought.. is it worth having a shared library of bash functions that are includes as part of the individual commands so that we don't get this repetition?? So new bash script called z390bash.sh -- and each script sources this file -- source z390bash.sh. This then makes any functions defined available to the bash script.. Again.. happy to approve, just a thought for next iteration.

Reply:
Agreed, a lot of repetition. Didn't know about the sourcing technique. Maybe one day. However, I like seeing everything in one place.